website/src/components/confetti/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useRef, useEffect } from 'react';
+import confetti from 'canvas-confetti';
+
+const ConfettiTrigger = ({ children }) => {
+  const triggerRef = useRef(null); // Use a ref to refer to the component's section
```

Comment:
I don't believe useRef is needed if we remove the event listeners (noted in another comment).

The only other benefit i can think of in using useRef is if we were positioning the confetti based on the location of the element on the page. Currently it looks like it's set to always be in the same vertical position `{ y: 0.6 }` and has its horizontal position set by the for loop `{ x: (i + 1) }`.

But based on how it's being used i think we can do away with the useRef entirely.

Admittedly i'm not the best with understanding `useRef` so i might be wrong here. But from testing it out it seems like it functions without it.